internal/dataplane/translator/ingressrules.go

```diff
@@ -105,6 +110,68 @@ func (ir *ingressRules) populateServices(
 	return serviceNamesToSkip
 }
 
+func (ir *ingressRules) handleBackendTLSPolices(
+	_ logr.Logger,
+	s store.Storer,
+	k8sService *corev1.Service,
+	failuresCollector *failures.ResourceFailuresCollector,
+	_ *ObjectsCollector,
```

Comment:
Why do we need the logger and objectsCollector if they are unused?

Reply:
leftover from previous implementation. Removing it